e2e/imgbuild/imgbuild.go

```diff
@@ -1941,6 +1950,104 @@ DEMO=demo=with===equals==signs
 	})
 }
 
+func (c imgBuildTests) buildUseExistingBuildkitd(t *testing.T) {
+	const (
+		bkLaunchTimeout = 10 * time.Second
+	)
+
+	tmpdir, tmpdirCleanup := e2e.MakeTempDir(t, "", "build_use_existing_buildkitd_e2e_", "dir")
+	t.Cleanup(func() {
+		if !t.Failed() {
+			tmpdirCleanup(t)
+		}
+	})
+
+	outputImgPath := filepath.Join(tmpdir, "image.oci.sif")
+
+	var cmd *exec.Cmd
+	var err error
+	var cmdPipe io.ReadCloser
+	e2e.Privileged(func(t *testing.T) {
```

Comment:
I don't think it's a great idea to launch a daemon as root from an e2e-test. If someone tries to run the test suite after building on a system where they may use buildkitd otherwise, we could cause issues.

I think it would be fair to run tests using one that's already running, or to start one rootless with a separate config file.